herbert_core/classes/MPIFramework/@parallel_config/private/check_and_set_cluster_.m

```diff
@@ -7,13 +7,11 @@
 % The cluster name (can be defined by single symbol)
 % or by a cluster number in the list of clusters
 %
-wrkr = which(obj.worker_);
-mff = MPI_clusters_factory.instance();
+    assert(~isempty(which(obj.worker)) || exist(obj.worker, 'file'), ...
```

Comment:
This code seems duplicated 3x is it worth making it a function?

Reply:
yeah... maybe... it's just it's in three different classes... :/